frame/staking/src/pallet/impls.rs

```diff
@@ -1245,13 +1245,12 @@ impl<T: Config> VoteWeightProvider<T::AccountId> for Pallet<T> {
 		// this will clearly results in an inconsistent state, but it should not matter for a
 		// benchmark.
 		let active: BalanceOf<T> = weight.try_into().map_err(|_| ()).unwrap();
-		let ledger = match Self::ledger(who) {
+		let mut ledger = match Self::ledger(who) {
 			None => StakingLedger::default_from(who.clone()),
```

Comment:
We expect benchmarks for the bags list to access the case where the ledger does not already exist (assuming the list is configured to use the staking pallet as the vote weight provider). AFAICT it was just an oversight that this did not set the `active` in the case we had to create a new ledger, but I want to make sure I understand why the benchmarks where working before in polkadot

Reply:
Found the regression: https://github.com/paritytech/substrate/commit/55cd07a7e22c26932f7bd16b87ea5a7569e38eb4#diff-d4fe039eee511c339d76db54bac9ed9da96b92c85efbe724ec1f805f2528acc7L1241-R1252